source/frontend/src/components/maps/leaflet/Control/LayersControl/data.ts

```diff
@@ -101,7 +101,7 @@ export const layersTree: ILayerItem[] = [
         format: 'image/png',
         zIndex: 10,
         id: 'firstNationsReserves',
-        color: '#ebe0d7',
+        color: '#DCBDAD',
         opacity: 0.9,
```

Comment:
would be nice to see a screenshot of the new colors in the layers menu. We usually attach screenshots to PR that involve user interface changes

Reply:
![Screenshot (9)](https://github.com/bcgov/PSP/assets/53246811/852dc8de-443d-4aca-bc8c-a13799649be8)
![Screenshot (10)](https://github.com/bcgov/PSP/assets/53246811/7d2a6621-396c-4371-9b9e-408ce2ea5d86)
![Screenshot (11)](https://github.com/bcgov/PSP/assets/53246811/f8dafa94-2d85-46b8-8a97-a9124e4eea99)
![Screenshot (12)](https://github.com/bcgov/PSP/assets/53246811/068721b2-5e01-4eb9-a819-629a59de5852)